### ISSUE TITLE
the results of camOdoCalib are different wtih different data of the same device!

### ISSUE THREAD

**Issue by yuyadanyadan** [hengli/camodocal#82]
one is the odometry position, while the postion of camera is from the ORB-SLAM2.
The result is not stable enough.maybe I make some mistakes, Can someone help me? thank you very much

**Comment by Unlingius** [comment]
"not stable" is completely divergence ?    What precision did you get? Im my experiments angles were +- 1 degree from ground-true values

**Comment by Unlingius** [comment]
@yuyadanyadan  what results angle difference you got?

**Comment by yuyadanyadan** [comment]
> @yuyadanyadan what results angle difference you got?

the results of angle difference is in 1 degree, while the translation difference may up to about 8cm.

**Comment by Unlingius** [comment]
@yuyadanyadan  thank you for your response. 1 degree is to huge for my purpose.. It would be interesting to check what angles precision will be if we will fix guest translation.

**Comment by yuyadanyadan** [comment]
@Unlingius  Do you research the calibration of odom and cameras? I also want to get more exact results, do you have any ideas?

**Comment by Unlingius** [comment]
yes, i have done my own method based on orbslam2 and without odometry, it process slam track only, it calculates angles relative to road plane with near 0.2 degree precision.  But it don't calculate translation. It hardly depends on fact that track is planar. Sorry, now i can't share it.

**Comment by yuyadanyadan** [comment]
Did you calibrate the extrinsic parameters of cameras and Wheeled encoder? And the angle error related to groundtruth is about 0.2 degree?

**Comment by Unlingius** [comment]
I did not use any additional sensors, only one camera. The result is angles relative to road plane. I use assumption, that in rest on horizontal plane angles relative to car frame are the same as angles relative to road.

**Comment by yuyadanyadan** [comment]
We do the different work. My work is calibrating the extrinsic parameters of camera and wheeled encoder.

**Comment by poaongithub** [comment]
same as mine, have you solved it? I would appreciate it if you would share your test dataset. @yuyadanyadan

**Comment by zhummming** [comment]
did you get the exact result? my results are not good and the same dataset has different results. can I try your dataset? @yuyadanyadan
